libmscore/measure.cpp

```diff
@@ -496,7 +496,7 @@ void Measure::layoutMeasureNumber()
       QString s;
       if (smn)
             s = QString("%1").arg(no() + 1);
-      int nn = 1;
+      unsigned nn = 1;
```

Comment:
why not `size_t`?

Reply:
because it leads to a different compiler warning (on a different compiler)